test/unit/utils.test.ts

```diff
@@ -525,4 +527,32 @@ describe('driver utils', function () {
       expect(isHello(doc)).to.be.false;
     });
   });
+
+  describe('class MongoDBNamespace', () => {
```

Comment:
I don't think it would hurt to add the happy path cases for sanity and clarity of expected normal behavior, since these are unit tests